### ISSUE TITLE
Marriage Certificate Step 3 Non-Descriptive Form Labels.mp4

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#152]
Screengrab/videos (insert file names here)
2020 08 25, Marriage Certificate Step 3 Non-Descriptive Form Labels.mp4

Link
https://registry.boston.gov/marriage

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOs 10.15.6 > Chrome 84

Trusted Tester Test
N/a

Brief Description
On the third step of the marriage certificate site, the form labels are not as descriptive as they are required
[2020 08 25, Marriage Certificate Step 3 Non Descriptive Form Labels.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5131442/2020.08.25.Marriage.Certificate.Step.3.Non.Descriptive.Form.Labels.mp4.zip)
 to be. 

Steps to reproduce
1)	Go through the steps and get to the third step of the Marriage Certificate site
2)	00:00 The ANDI output for Month and Year does not provide all the information. Specifically, the Names of the Section are not label shown by ANDI.

Desired Result
•	The form fields have labels descriptive enough to understand the purpose of the form fields

Actual Result
•	The form fields are just labeled ‘date 1 Month’, ‘date 1 Month’, … . That does not give all the information relevant for the form field. The text for the ANDI output for those fields should be: ‘Please Enter a date range date 1 Month’, …
